File: Swift/0001.swift
import Foundation

// Function to calculate the sum of multiples of 3 or 5 below a given limit
func sumOfMultiples(below limit: Int) -> Int {
    var totalSum = 0
    for i in 1..<limit {
        if i % 3 == 0 || i % 5 == 0 {
            totalSum += i
        }
    }
    return totalSum
}

// Define the limit
let limit = 1000

// Calculate the result
let result = sumOfMultiples(below: limit)

// Print the result
print("The sum of all multiples of 3 or 5 below \(limit) is: \(result)")

File: Ruby/0001.rb
# Define a method to calculate the sum of multiples of 3 or 5 below a given limit
def sum_of_multiples(limit)
    total_sum = 0
    (1...limit).each do |i|
      if i % 3 == 0 || i % 5 == 0
        total_sum += i
      end
    end
    total_sum
  end
  
  # Define the limit
  limit = 1000
  
  # Calculate the result
  result = sum_of_multiples(limit)
  
  # Print the result
  puts "The sum of all multiples of 3 or 5 below #{limit} is: #{result}"
  